components/apimgt/org.wso2.carbon.apimgt.tracing/pom.xml

```diff
@@ -69,6 +69,11 @@
             <artifactId>org.apache.felix.scr.ds-annotations</artifactId>
             <scope>provided</scope>
         </dependency>
+        <dependency>
+            <groupId>io.opentracing</groupId>
+            <artifactId>opentracing-api</artifactId>
+            <version>0.31.0</version>
```

Comment:
is this change intentionally to this pr.

Reply:
yes. Without this dependancy, I get a compilation error